sccp/figures/figureS2.py

```diff
@@ -30,29 +27,22 @@ def makeFigure():
     subplotLabel(ax)
 
     # Import of data
-    data, cell_types, _ = load_lupus_data()  # don't need to get patient color mappings
-    rank = 30
-    cellState = 28
-    cmp = 28
-
-    # run pf2
-    _, factors, projs, _ = parafac2_nd(
-        data,
-        rank=rank,
-        random_state=1,
-    )
-
-    projs = np.concatenate(projs, axis=0)
+    _, cell_types, _ = load_lupus_data()  # don't need to get patient color mappings
+    rank = 39
+    cellState = 11
+    cmp = 11
+
+    _, factors, projs, = openPf2(rank = rank, dataName = 'lupus', optProjs=True)
+
+
+    proj_B = projs @ factors[1]
```

Comment:
replace running pf2 with openPf2 (note that savePf2 concatenates the projections; so we don't need that step again here)